vtex/ab-tester
# Custom parameters of initialization

#### What is the purpose of this pull request?
Allow custom configuration of duration of initial stage and the proportion of traffic in each workspace during this part of the test.

#### What problem is this solving?
We regularly faced the problem of customize the way the initial stage of the test runs, i.e. how long it takes and the proportion of traffic directed to each workspaces. This PR creates a new route that configures custom parameters that aims to allow these configurations.

#### How should this be manually tested?
It's funny but this is being tested in production with Boticario. Just start a test with custom parameters and see if we have the desired behavior.

#### Types of changes
